_observing-your-data/alerting/comp-monitors.md

```diff
@@ -0,0 +1,444 @@
+---
+layout: default
+title: Composite monitors
+nav_order: 3
+parent: Alerting
+has_children: false
+redirect_from:
+---
+
+# Composite monitors
+
+---
+
+#### Table of contents
+- TOC
+{:toc}
+
+---
+
+## About composite monitors
+
+Basic [monitor types]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/monitors/#monitor-types) for the Alerting plugin are designed to define a single trigger type. For example, a document level monitor can trigger an alert based on a query's match with documents, while a bucket level monitor can trigger an alert based on queries aimed at aggregated values in a data source. The composite monitor extends the usefulness of the individual monitor types by providing functionality that can chain the output of multiple monitor types into a single workflow that includes multiple triggers and permits analysis of a data source based on multiple criteria. This allows you to derive more granular information about a data source, and it doesn't require you to manually coordinate the scheduling of the separate monitors.
+
+Composite monitors solve limitations with basic monitors in the following ways:
+
+* Composite monitors give you the ability to create complex queries through a combination of triggers generated by multiple types of monitors.
+* They have the capacity to define a pipeline of rules and queries that are run as a single execution.
+* They can chain together individual monitors so that the output of one can act as the input for the next, repeating this through the entire sequence of the execution.
+* They provide a more complete view of a given data source by running multiple monitors and multiple types of monitors in sequence, creating finer focused results and reducing noise in the results.
+
+
+## Key terms
+
+The key terms in the following table describe the basic concepts behind composite monitors. For additional terms common to all types of monitors, see [Key terms]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/monitors/#key-terms) for basic monitors.
+
+| Term | Definition |
+| :--- | :--- |
+| Composite monitor | A combination of multiple individual monitors that includes functionality to execute each monitor in a sequence. The compound monitor can query different aspects of a dataset based on the type of monitors included in the composite monitor's definition.  |
+| Delegate monitor | Any monitor of any monitor type used in the chained workflow of a compound monitor. Delegate monitors are executed sequentially according to their order in the monitor definition. |
+| Findings chain | A sequence of findings where the findings for each monitor are used as the inputs for subsequent monitors. 
+| Alert chain | A sequence of alerts where an alert for each monitor is used as the input for a subsequent monitor.  |
```

Comment:
this needs to be fixed.
Terms are 

1. chained alert: Chained alerts are generated from composite monitor triggers when delegate monitors generate alerts The chained alert trigger condition supports the use of the logical operators AND, OR, and NOT, so you can combine multiple functions into a single expressions.

2. Audit Alert: Delegate monitors generate alerts in AUDIT state. Users won't be notified about each individual audit alerts and don't need to acknowledge them. Audit alerts will be used to evaluate chained alert trigger conditions in composite monitors.

Reply:
Got it. Thanks for the additions. Finishing touches are in process.